### ISSUE TITLE
[1.9.23] iPhone 6 Plus / 10.0.1, iPad Mini 3 / 9.3.5 - "Cancel" pop up message is not disappearing when user navigated to "Create" screen.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#716]
Frequency: Every time

Action:
1: Launch the app.
2: Tap on "START" and select any motifs from any category.
3: Tap on any music from "Find Music" tab.(Loading process will be displayed)
4: Tap on "Next" option while loading the music.("Cancel" pop up will displayed)
5: Wait till the loading process complete and user is navigated to "Create" screen.
6: Check if "Cancel" pop is not disappearing.

Expected result: "Cancel" pop up message should get disappeared when user proceeds to "Create" screen.

Actual result: 
1: "Cancel" pop up message is not disappearing when user navigated to "Create" screen.
2: Music is not getting removed when user taps on "OK" option "Cancel" pop up on "Create" screen; previously added music is playing.

Refer screenshot for more details.

**Comment by testershub** [comment]
![cancel_pop up](https://cloud.githubusercontent.com/assets/18608449/18699406/e68675c6-7fee-11e6-879d-2a396cae5b98.jpg)

**Comment by vysohanych** [comment]
@xiuie I can't reproduce this issue on build 1.9.23 on iPhone 5S / iOS 9.3.5

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.9.23] on iPhone 6 / iOS 10.0.1

But This is reproducible for Find Music song not for My Music. Because My Music song immediately loads (no loading process occurs in My Music song)

**Comment by testershub** [comment]
@xiuie @hiteshbheda My mistake. It's reproducible for music under "Find Music" tab.

**Comment by hiteshbheda** [comment]
@testershub @xiuie Yes, Agree. I have already mentioned that in above comments. Thanks @testershub
